### PR TITLE
Uritools recipe

### PR DESCRIPTION
**Pull request by ltalirz**
conda recipe for the [uritools](https://pypi.python.org/pypi/uritools) package.

This is my first conda recipe, suggestions are welcome!

### CODE REVIEW

File: recipes/uritools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python 2.7*|3.3*|3.4*|3.5*
```
Comment (synapticarbors) [inline recipes/uritools/meta.yaml]: Just specify `python` without the versions.

File: recipes/uritools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - ipaddress # [py27]
+
+test:
+  imports:
+    - uritools
+    - ipaddress # [py27]
```
Comment (synapticarbors) [inline recipes/uritools/meta.yaml]: You don't need to import `ipaddress` in the `test` section.

File: recipes/uritools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{% set version = "2.0.0"%}
+{% set name = "uritools" %}
+{% set md5 = "0142c6a035df53b37bca67ce172beb50" %}
```
Comment (synapticarbors) [inline recipes/uritools/meta.yaml]: Please use the `sha256` hash.

File: recipes/uritools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
```
Comment (synapticarbors) [inline recipes/uritools/meta.yaml]: `sha256`

File: recipes/uritools/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+#  patches:
```
Comment (synapticarbors) [inline recipes/uritools/meta.yaml]: Delete comments.

File: recipes/uritools/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+    - setuptools
+
+  run:
+    - python 2.7*|3.3*|3.4*|3.5*
+    - ipaddress  # [py27]
+
+test:
+  imports:
+    - uritools
+    - ipaddress  # [py27]
```
Comment (synapticarbors) [inline recipes/uritools/meta.yaml]: No need to import `ipaddress` here.

File: recipes/uritools/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    - python -m unittest discover
+
+  source_files:
+    - tests
+
+about:
+  home: https://github.com/tkem/uritools/
+  license: MIT
+  license_family: MIT
+  license_files: LICENSE
```
Comment (synapticarbors) [inline recipes/uritools/meta.yaml]: This should be `license_file` not `license_files`.

**Comment by jakirkham** [comment]
Anything else @synapticarbors?

**Pull request merged**